### PR TITLE
Created header footer

### PR DESCRIPTION
**Pull request by adodge-bcg**
# Description

Created header and footer base components, added images folder and imported images used on header, apple icon, and tab icon.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Ran tests localy, created storybook stories.
Header
Footer

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## For bcgov contributors:

this PR fixes jira ticket: SPDBCSC-127

### CODE REVIEW

File: src/ecrc-frontend/src/components/base/footer/Footer.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+              </a>
+            </li>
+            <li className="nav-item">
+              <a
+                className="nav-link"
+                href="https://www2.gov.bc.ca/gov/content/home/copyright"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                Copyright
```
Comment (sdevalapurkar-bcgov) [inline src/ecrc-frontend/src/components/base/footer/Footer.js]: are all of these values gonna be static? or should they be props?

File: src/ecrc-frontend/src/components/base/footer/Footer.stories.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+import Footer from "./Footer";
+
+export default {
```
Comment (sdevalapurkar-bcgov) [inline src/ecrc-frontend/src/components/base/footer/Footer.stories.js]: we dont need this -> let's use `storiesOf`, it makes the stories cleaner

File: src/ecrc-frontend/src/components/base/header/Header.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+export default function Header({ name }) {
```
Comment (sdevalapurkar-bcgov) [inline src/ecrc-frontend/src/components/base/header/Header.js]: let's use the same pattern discussed earlier:
```suggestion
export default function Header({ header: { name } }) {
```

File: src/ecrc-frontend/src/components/base/header/Header.stories.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+import Header from "./Header";
+
+export default {
```
Comment (sdevalapurkar-bcgov) [inline src/ecrc-frontend/src/components/base/header/Header.stories.js]: same here, dont need this and can clean it up by using `storiesOf`

File: src/ecrc-frontend/src/components/base/footer/Footer.js (hunk 1 above)
Comment (adodge-bcg) [reply src/ecrc-frontend/src/components/base/footer/Footer.js]: Im not even sure any more. They are on all public facing gov pages I looked at, but they only appear to be on the home/early pages of cannabis and pillpress and are absent later. When they are present they are static and the same throughout.

File: src/ecrc-frontend/src/components/base/footer/Footer.stories.js (hunk 2 above)
Comment (adodge-bcg) [reply src/ecrc-frontend/src/components/base/footer/Footer.stories.js]: Can do, I just picked CSF as it is the newer feature of storybook and they recommended switching from the storiesOf format. https://storybook.js.org/docs/formats/storiesof-api/

File: src/ecrc-frontend/src/components/base/footer/Footer.js (hunk 1 above)
Comment (sdevalapurkar-bcgov) [reply src/ecrc-frontend/src/components/base/footer/Footer.js]: let's leave them static for now - sounds good

File: src/ecrc-frontend/src/components/base/footer/Footer.stories.js (hunk 2 above)
Comment (sdevalapurkar-bcgov) [reply src/ecrc-frontend/src/components/base/footer/Footer.stories.js]: cool, hadn't looked into it but after reading about it, it seems like a better option which allows better portability and testing -> let's keep it this way then and use CSF, thanks!

File: src/ecrc-frontend/src/components/base/header/Header.stories.js (hunk 4 above)
Comment (sdevalapurkar-bcgov) [reply src/ecrc-frontend/src/components/base/header/Header.stories.js]: there seems to be a module that can convert from `storiesOf` to `csf` directly so we can use whatever version we want and are more comfortable with for now - and then convert them later

File: src/ecrc-frontend/src/components/base/header/Header.js (hunk 3 above)
Comment (adodge-bcg) [reply src/ecrc-frontend/src/components/base/header/Header.js]: Fixed

**Pull request merged**